server/tests/test_get_symbol.rs

```diff
@@ -150,6 +150,72 @@ fn test_hover_on_model_field_and_method() {
     );
 }
 
+#[test]
+fn test_hover_on_namespace_and_module() {
```

Comment:
The reason I placed this test here is the similarities with the test above (test_hover_on_model_field_and_method).
But should we have a test file per feature, both tests would end up in a test_hover file.